lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
+    const isAudit = 'result' in auditOrCategory;
+    const tagName = isAudit ? 'details' : 'div';
+
+    // Score template. Audits get a details dropdown. Categories don't.
+    const tmpContainer = this._createElement('div');
+    tmpContainer.innerHTML = `<div class="lighthouse-score">
```

Comment:
Don't be scared. Completely safe since we control all inputs for the initial set.